lib/truffle/date.rb

```diff
@@ -1378,7 +1378,7 @@ def <=> (other)
       begin
         l, r = other.coerce(self)
         return l <=> r
-      rescue NoMethodError
+      rescue NoMethodError # rubocop:disable Lint/HandleExceptions
```

Comment:
A nicer way to avoid the warning here and in the rest of this file would be to move the `nil` in the `rescue` and remove extra `return`. But it's a larger change.